src/css/docs.css

```diff
@@ -167,11 +171,27 @@ table tr td + td + td:last-child {
 }
 
 /* Search */
-.navbar .navbar__search .navbar__search-input {
-  width: 15.5rem;
+.DocSearch.DocSearch-Button {
+  width: 12.5rem;
+  height: 2.25rem;
   font-size: 1rem;
+  border-radius: 0;
+  color: #ffffff;
+  background-color: #9a2829;
   font-family: var(--ifm-font-family-base);
-  border-radius: var(--ifm-global-radius);
+}
+.DocSearch.DocSearch-Button:hover {
+  background-color: #9a2829;
+  color: #ffffff;
+  height: 2.25rem;
+}
+.navbar__items .DocSearch-Button .DocSearch-Search-Icon {
+  width: 1rem;
+  height: 1rem;
+  color: #ffffff;
+}
```

Comment:
There's a bunch of issues here, but I'm just going to push a fix.

Reply:
Specifically, you should have been using the css variables and reviewing the different states, hover/active/etc.